src/plots/gl2d/convert.js

```diff
@@ -68,6 +68,7 @@ function Axes2DOptions(scene) {
         [0, 0, 0, 0.5]
     ];
     this.gridLineWidth = [1, 1];
+    this.gridLineDash = ['', ''];
```

Comment:
This is not going to work for `gl2d`.
Please undo changes to `src/plots/gl2d/convert.js`.